### ISSUE TITLE
Implement Login Page

### ISSUE THREAD

**Issue by keymane** [OpenSRP/opensrp-client-anc#4]
Final mockup below:

<img width="413" alt="screen shot 2018-07-06 at 8 04 24 am" src="https://user-images.githubusercontent.com/3488986/42360754-63907aec-80f3-11e8-9395-07d4235c03ab.png">

- [x] User interface
- [x] Authentication Functionality
- [x] Conversion to MVP pattern
- [x] Unit Tests

**Comment by ndegwamartin** [comment]
@rowo might we wanna include the version and build timestamp info?

**Comment by cagulas** [comment]
@ndegwamartin Yes, please include. It was an oversight on our part.

**Comment by cagulas** [comment]
This passes QA (V1.0 built on 2 Aug 2018). Looks great!

**Comment by rowo** [comment]
@ndegwamartin Is there a way to make sure the "show password" checkbox is visible when the keyboard is up while typing in the password?

**Comment by ndegwamartin** [comment]
@rowo not quite, but there is a workaround solution. I've updated the implementation, so previously when the keyboard was showing one couldn't scroll the login view to make the show password widget visible. Now you can scroll to it while typing.

**Comment by cagulas** [comment]
Sorry, spoke too soon. Couple of minor things:

- [ ] Let's abbreviate the month in the build date.
- [ ] The check mark in the show password box is very blurry.
